### PR TITLE
Move travis to 18.04

### PR DESCRIPTION
**Pull request by Karmastic**
We want to start building and testing devnet (nightly) on Ubuntu 18.04 so we can move production to it after more testing.
Remove building of debian package - it's now done for release builds only.
Remove GATE_PREFIX - we'll use separate buckets for the same purpose.

NOTE: This will fail to install and run on CentOS due to an obsolete glibc version in CentOS.

### CODE REVIEW

File: scripts/build_packages.sh [hunk 1]
```diff
@@ -106,30 +99,16 @@ for var in "${VARIATION_ARRAY[@]}"; do
...
         if [ $? -ne 0 ]; then
             echo "Error creating tar file for package ${PLATFORM}.  Aborting..."
             exit 1
         fi
 
         cd ${PLATFORM_ROOT}/tools
-        tar -zcf ${PKG_ROOT}/${GATE_PREFIX}tools_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.tar.gz * >/dev/null 2>&1
+        tar -zcf ${PKG_ROOT}/tools_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.tar.gz * >/dev/null 2>&1
         popd
-
```
Comment (algobolson) [inline scripts/build_packages.sh]: build_release.sh calls build_packages.sh and expects that to build the .deb
we could turn it off here with a variable test, e.g. `if [ ! -z "${BUILD_DEB}" ];` or so

**Comment by tsachiherman** [comment]
I think that our goal should be expanding our supported platform base rather than shrinking it..
What would it take to have a "special build" for RedHat/CentOS builds ?

**Comment by algobolson** [comment]
ubuntu 16.04 is supposed to get maintenance upgrades through 2020 ( https://ubuntu.com/about/release-cycle ), but _we_ don't have to support it that long. Moving to newer systems makes things easier for us because they have newer versions of things installed with bug fixes and nicer features. The release build should continue to support Ubuntu 18.04 and Centos 7.

**Comment by Karmastic** [comment]
> I think that our goal should be expanding our supported platform base rather than shrinking it..
> What would it take to have a "special build" for RedHat/CentOS builds ?

I'm going to exercise our RPM package for R1 and R2, our CentOS machines.  They'll be supported - they just won't support the installer (unless the user wants to build glibc for CentOS 7 themselves)

File: scripts/build_packages.sh [hunk 2]
```diff
@@ -128,7 +121,7 @@ for var in "${VARIATION_ARRAY[@]}"; do
                 exit 1
             fi
             pushd ${DEBTMP}
-	    cp -p *.deb ${PKG_ROOT}/${GATE_PREFIX}algorand_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.deb
+            cp -p *.deb ${PKG_ROOT}/${GATE_PREFIX}algorand_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.deb
```
Comment (algobolson) [inline scripts/build_packages.sh]: one more GATE_PREFIX

**Review by algobolson — CHANGES_REQUESTED** [review]

**Review by algobolson — APPROVED** [review]

**Pull request merged**